cpp/src/arrow/util/spaced_sse_generated.h

```diff
@@ -0,0 +1,608 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// Automatically generated file; DO NOT EDIT.
+
+#pragma once
+
+namespace arrow {
+namespace internal {
+
+static constexpr uint8_t kMask128SseCompressEpi32[] = {
```

Comment:
Is it useful to have the definitions in a `.h` file rather than `.cc`? Those tables are a bit large and it would be better not to duplicate them around.

Reply:
Good catch, will move to cc file